src/utils/ucc_time.h

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (C) Mellanox Technologies Ltd. 2021.  ALL RIGHTS RESERVED.
+ * See file LICENSE for terms.
+ */
```

Comment:
why can't we use ucs/time/time.h ?

Reply:
it is not exposed as public header. only time_def.h is. time.h contains arch specific implementation.